scrapyjs/middleware.py

```diff
@@ -96,20 +108,31 @@ def process_request(self, request, spider):
             # are not respected.
             headers=Headers({'Content-Type': 'application/json'}),
         )
-
         self.crawler.stats.inc_value('splash/%s/request_count' % endpoint)
         return req_rep
 
     def process_response(self, request, response, spider):
-        splash_options = request.meta.get("_splash_processed")
+        splash_options = self.get_splash_options(request, spider)
         if splash_options:
             endpoint = splash_options['endpoint']
             self.crawler.stats.inc_value(
                 'splash/%s/response_count/%s' % (endpoint, response.status)
             )
-
+            response = self.html_response(response, request)
```

Comment:
> I also think that by default response should be returned without changes, but if we will have option to enable html response and use scrapyjs transparently i.e. getting rendered html without handling this in callback

sure we can make it optional and just make it conditional on existence of some key either in meta or in spider attributes. What do you think @kmike ?